### PR TITLE
Add anchor to configuration options in documentation

### LINKED ISSUE #533: Add anchor to configuration options in documentation
Thank you for taking your time to talk with us!

**What is this issue about?**

- [ ] Bug report
- [x] Feature request
- [ ] Question

**Description**
Is not possible to share a pointer to a specific configuration option using a url. That's cumbersome because the page is relatively long so users need to navigate and then search for it by name.

### PR DESCRIPTION
**Pull request by uniqueck**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Refactor
- [ ] Build improvement
- [ ] Other (please describe)

**What is the goal of this pull request?**
Fixes #533

**Are there any alternative ways to implement this?** 
no, but we can discuss how to define the id's for the anchors 

**Are there any implications of this pull request? Anything a user must know?**
no

**Is it related to an existing issue?**
 
- [x] Yes
- [ ] No

*Finally, please add a corresponding entry to CHANGELOG.adoc*

### CODE REVIEW

**Comment by uniqueck** [comment]
@abelsromero can you review this PR?

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -13,6 +13,10 @@ For a detailed view of what has changed, refer to the {uri-repo}/commits/main[co
 
 == Unreleased
 
+Documentation::
+
+  * add id's for all parameters of process-asciidoc, to be able to generate direct urls (https://github.com/uniqueck[@uniqueck]) (#533)
```
Comment (abelsromero) [inline CHANGELOG.adoc:18]: Capitalize A in add?

Comment (abelsromero) [reply CHANGELOG.adoc:18]: Also, you updated all configurations for all goals, not only `process-asciidoc` :+1: that deserves some credit

File: docs/modules/plugin/partials/auto-refresh-mojo-parameters.adoc [hunk 2]
```diff
@@ -1,6 +1,8 @@
+[[interval]]
```
Comment (abelsromero) [inline docs/modules/plugin/partials/auto-refresh-mojo-parameters.adoc:1]: I admit I have become more acostumed to the short notation `[#interval]`. https://docs.asciidoctor.org/asciidoc/latest/attributes/ids/#anchor 
I know it's nitpicking, but it allows more flexibility so we don't need to think about the limitations of the other notation. Would you mind using it? :pleading_face:

**Comment by abelsromero** [comment]
> @abelsromero can you review this PR?

Thanks for the reminder!

File: docs/modules/plugin/partials/auto-refresh-mojo-parameters.adoc (hunk 2 above)
Comment (uniqueck) [reply docs/modules/plugin/partials/auto-refresh-mojo-parameters.adoc:1]: ok no problem, I will change it.

File: CHANGELOG.adoc (hunk 1 above)
Comment (uniqueck) [reply CHANGELOG.adoc:18]: You are so right ;)

**Comment by abelsromero** [comment]
Thanks!! :partying_face:

**Pull request merged**